src/core/core.scale.js

```diff
@@ -594,7 +597,18 @@ module.exports = function(Chart) {
 						context.font = tickLabelFont;
 						context.textAlign = (isRotated) ? "right" : "center";
 						context.textBaseline = (isRotated) ? "middle" : options.position === "top" ? "bottom" : "top";
-						context.fillText(label, 0, 0);
+						
+						lineHeight = context.measureText("M").width * 1.2;
```

Comment:
What's the reason for measuring a character here? On line 326, the height is `(tickFontSize * numLines) + (lineSpacing * (numLines - 1))`. Shouldn't line height here be `tickFontSize + lineSpacing`? Or am I missing something?